doc/lua_api.txt

```diff
@@ -1837,6 +1837,9 @@ to games.
 
 * `disable_repair`: If set to 1 for a tool, it cannot be repaired using the
   `"toolrepair"` crafting recipe
+* `speed`: Changes movement speed for players and items will moving on the node.
+	Only a decrease in speed is possible.
+   * `speed = -1`: to slow down the movement (min. `-100`, unable to walk)
```

Comment:
i like the feature but i'd prefer if speed is a percentage, 100% = normal 0% = unable to walk 1000% = 10x the normal speed. It's better in terms of understanding that having complicated calculation.  speedFactor = uint16 and that's all, no divide, no addition, no clamp, just a raw value factor value

Reply:
Unspecified groups default to 0, hence it would require special treatment for a specific group.

Reply:
Why should only a decrease be possible? Sounds arbitrary to me.